src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/conda-store/worker.tf

```
@@ -94,10 +94,12 @@ resource "kubernetes_deployment" "worker" {
             required_during_scheduling_ignored_during_execution {
               node_selector_term {
                 match_expressions {
+                  // Since all node selectors use the same provider key, we can reuse
+                  // the same key even when overriding the value.
                   key      = var.node-group.key
                   operator = "In"
                   values = [
-                    var.node-group.value
+                    var.worker-overrides.node_selector != null ? var.worker-overrides.node_selector : var.node-group.value
```

```suggestion
                    var.node-group.value
```
This does not work the way it is, I am putting it back